test/client/fixtures/sandbox/event/event-simulator-test.js

```diff
@@ -475,6 +476,149 @@ if (!browserUtils.isFirefox) {
     });
 }
 
+test('mouse event buttons properties', function () {
+    var actualLog = {};
+
+    var getButtonsProperty = function (buttons) {
+        return browserUtils.isSafari ? void 0 : buttons;
+    };
+
+    var eventHandler = function (e) {
+        var args = {
+            type:    e.type,
+            button:  e.button,
+            buttons: getButtonsProperty(e.buttons),
+            which:   e.which
+        };
+
+        actualLog[e.type] = args;
+    };
+
+    var getArgs = function (type, button, buttons, which) {
+        return {
+            type:    type,
+            button:  button,
+            buttons: getButtonsProperty(buttons),
+            which:   which
+        };
+    };
+
+    var getExpectedMouseMoveEvents = function (type) {
+        return [{
```

Comment:
I prefer this style:
```js
return [
    {
         foo: bar,
         foo: bar,
    },
    {
         foo: bar,
         foo: bar,
    }
];
```
What do ypu think?

Reply:
if it's clearer, ok